src/data.js

```diff
@@ -892,6 +892,32 @@ let Worker = Entity.configure({
 
     return item;
   },
+}).configure({
+  version:            4,
+  properties: {
+    provisionerId:    Entity.types.String,
+    workerType:       Entity.types.String,
+    workerGroup:      Entity.types.String,
+    workerId:         Entity.types.String,
+    /**
+     * List of objects with properties:
+     * - taskId
+     * - runId
+     * See JSON schema for documentation.
+     */
+    recentTasks:      Entity.types.JSON,
+    disabled:         Entity.types.Boolean,
+    // the time at which this worker should no longer be displayed
+    expires:          Entity.types.Date,
+    firstClaim:       Entity.types.Date,
+  },
+  migrate(item) {
+    item.firstClaim = new Date(2000, 0, 1);
+    item.recentTasks = [];
+    item.disabled = false;
```

Comment:
Only `recentChanges` needs to be updated here -- the other two fields have not changed.